tests/software/kallistobustools/ref/main.nf

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { KALLISTOBUSTOOLS_REF } from '../../../../software/kallistobustools/ref/main.nf' addParams( options: [args:"standard"] )
+include { KALLISTOBUSTOOLS_REF as KALLISTOBUSTOOLS_REF_LAMANNO } from '../../../../software/kallistobustools/ref/main.nf' addParams( options: [args:"lamanno"] )
+include { KALLISTOBUSTOOLS_REF as KALLISTOBUSTOOLS_REF_NUCLEUS } from '../../../../software/kallistobustools/ref/main.nf' addParams( options: [args:"nucleus"] )
+
+workflow test_kallistobustools_ref_standard {
+
+    input = [ [id:'test.standard'], // meta map
+        [file("${launchDir}/tests/data/delete_me/kallistobustools/GRCm39.chr19_100k.fa.gz", checkIfExists: true)]
```

Comment:
We need to remember to actually delete all of this custom test data. I am wondering whether its better to just add it to another repo somewhere so we don't bloat the git history of nf-core/modules 🤔 We should maybe try and get that human data PR in as soon as (Ping @FriederikeHanssen)

Reply:
I have actually added some of this test-data to nf-core test-datasets:
https://github.com/nf-core/test-datasets/pull/231

I just didn't know how to correctly use data that is part of the test-datasets repo in these tests without having them inside the modules repo?!

Reply:
Ah good point ... we could either make a branch in nf-core/test-datasets for temporary test data or make a whole repo that we could monitor and later delete if necessary 🤔 
@FloWuenne if you have a look at the `test.config` profiles (in the conf folder) of any published pipeline, it shows you how you can just link to a file somewhere on github.